contracts/IdentityManager.sol

```diff
@@ -208,4 +208,12 @@ contract IdentityManager {
         t := eq(a, and(mask, calldataload(4)))
     }
   }
+
+  function isOwner(address identity, address user) constant returns (bool) {
```

Comment:
User might be a little misleading langauge @oed. Maybe *key* (as below) would be better?